src/ott/initializers/linear/initializers.py

```diff
@@ -313,3 +338,90 @@ def body_fn(i: int, f: jnp.ndarray) -> jnp.ndarray:
     return f.at[i].set(new_f)
 
   return jax.lax.fori_loop(0, len(f), body_fn, f)
+
+
+@jax.tree_util.register_pytree_node_class
+class SubsampleInitializer(DefaultInitializer):
+  """Subsample initializer :cite:`thornton2022rethinking:22`.
+
+  Subsample each :class:`~ott.geometry.pointcloud.PointCloud`, then compute
+  :class:`Sinkhorn potential <ott.problems.linear.potentials.DualPotentials>` from the
+  subsampled approximations and use this potential to initialize Sinkhorn potentials/scalings
+  for the original problem.
+
+  Args:
+    subsample_n: number of points to subsample from each :class:`~ott.geometry.pointcloud.PointCloud`.
+  """
+
+  def __init__(
+      self,
+      subsample_n: int,
+      subsample_n_y: Optional[int] = None,
+      sinkhorn_kwargs: Optional[Dict[str, Any]] = None,
```

Comment:
I think we can pass this as `**kwargs: Any` (or `**sinkhorn_kwargs: Any`), no need to do `sinkhorn_kwargs or {}` later.

Reply:
This is consistent with https://github.com/ott-jax/ott/blob/57f13afe5503c33c495a1052eaf36292d71ab2a8/src/ott/solvers/linear/sinkhorn.py#L736

I am not sure of a better way of doing it. I don't think **sinkhorn_kwargs would work as need to store kwargs then re-use later